Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -3730,6 +3730,18 @@ void GLContext::get_material_param(Face face, GLenum pname, T* params)
     }
 }
 
+void GLContext::gl_copy_tex_sub_image_2d(GLenum target, GLint level, GLint xoffset, GLint yoffset, GLint x, GLint y, GLsizei width, GLsizei height)
+{
+    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_copy_tex_sub_image_2d, target, level, xoffset, yoffset, x, y, width, height);
+    RETURN_WITH_ERROR_IF(!(target == GL_TEXTURE_2D), GL_INVALID_ENUM);
```

Comment:
`target != GL_TEXTURE_2D`?

Reply:
Yep.

```
GL_INVALID_ENUM is generated if target is not GL_TEXTURE_2D, GL_TEXTURE_CUBE_MAP_POSITIVE_X, GL_TEXTURE_CUBE_MAP_NEGATIVE_X, GL_TEXTURE_CUBE_MAP_POSITIVE_Y, GL_TEXTURE_CUBE_MAP_NEGATIVE_Y, GL_TEXTURE_CUBE_MAP_POSITIVE_Z, GL_TEXTURE_CUBE_MAP_NEGATIVE_Z, GL_TEXTURE_1D_ARRAY, or GL_RECTANGLE.
```

Reply:
Whoops. I had a few more, but I realised they were _very_ exotic enum values, so I forgot to modify this. I'm actually missing `GL_TEXTURE_1D_ARRAY` so I'll add that.